lib/sharedTypes.js

```diff
@@ -40,6 +40,51 @@ export const LocationType = new GraphQLObjectType({
   },
 });
 
+const statusEnum = new GraphQLEnumType({
+  name: 'Status',
+  description: 'One of the films in the Star Wars Trilogy',
+  values: {
+    PRE_RELEASE: {
+      value: 'PRE_RELEASE',
+      description: 'The status before the tickets have been released',
+    },
+    RELEASED: {
+      value: 'RELEASED',
+      description: 'The status once the tickets have been released',
+    },
+    SOLD_OUT: {
+      value: 'SOLD_OUT',
+      description: 'The status once the tickets have all been sold out',
+    },
+    LIVE: {
+      value: 'LIVE',
+      description: 'The status when the event is currently happening',
+    },
+    ENDED: {
+      value: 'ENDED',
+      description: 'The status when the event has ended',
+    },
+  },
+});
+
+export const StatusType = new GraphQLObjectType({
+  name: 'StatusType',
+  fields: {
+    enum: {
+      type: statusEnum,
+      description: 'The status of the event',
+    },
+    link: {
+      type: GraphQLString,
+      description: 'The action of the event status button',
```

Comment:
There shouldn't be mentions of buttons or any other UI elements at this stage - Badger Brain is all about pure data